### DISCUSSION TITLE
Can we add a new feature that makes suggestions?

### DISCUSSION THREAD

**Discussion by Sebastian-pz** [Sebastian-pz/ducker-api#75] [Q&A] [answered]
Suggest users and cuacks

**Comment by mgstraface** [comment, marked as answer]
Creo que deberíamos hacerlo con la temática de los cuacks.

**Reply by Sebastian-pz** [reply]
Me parece bien, utilizar palabras claves para identificar que contenido es relevante para determinados usuarios 👍🏻
